### DISCUSSION TITLE
Change loading screen background

### DISCUSSION THREAD

**Discussion by davidmaruscsak00** [cables-gl/cables_docs#300] [Answers & Questions]
Hello,

I am looking for a way to change the cables.gl default loading screen.

<img width="1172" alt="Screenshot 2022-07-26 at 10 58 29" src="https://user-images.githubusercontent.com/23195286/180967487-60bbc118-db5e-40e2-b4e4-894497ec4f38.png">

Now it has a black background, is there a way to change its color?

Thanks :)

**Comment by tastez** [comment]
Hi - you can use LoadingStatus OP to open a gate to another part of a patch that is super light - for example an animation or simply a background color - then as your content loads, the gate will be opened by the Loading Status op and show the animation or a background color. Once loading is finished, the gate will close and you can hide the loading screen. There is also cool things you can do with the trigger and starting/stopping animations - but hopefully this gets you started
![image](https://user-images.githubusercontent.com/78642112/180973017-3b8e762c-89cf-4ac5-ac82-8da8cf69c0a9.png)

**Reply by davidmaruscsak00** [reply]
Thanks for the suggestion. I have tried it. It looks like there are two loading steps.
Now when loading the patch starts in black then after a few secs it turns to the color I set using your suggestion.

Do you know a way to affect the initial black loading screen?

here is a link to the patch:
https://cables.gl/p/IrjD_1

**Reply by pandrr** [reply]
its the background color of the website

**Reply by davidmaruscsak00** [reply]
No, its always black. I tried it embedded on a website with grey bg color.

**Reply by pandrr** [reply]
try to set alpha in mainloop and put the clearcolor way up under mainloop

**Reply by steam0r** [reply]
there is also some information in the docs regarding this: https://cables.gl/docs/faq/embedding/transparent_canvas/transpcanvas
